fetch_moveit_config/launch/move_group.launch

```diff
@@ -16,6 +16,7 @@
   <!-- move_group settings -->
   <arg name="allow_trajectory_execution" default="true"/>
   <arg name="allow_active_sensing" default="false"/>
+  <arg name="moveit_octomap_sensor_params_file" default="$(find fetch_moveit_config)/config/sensors.yaml" />
```

Comment:
I just added the default here @cjds

Reply:
So there will be no error when running `roslaunch fetch_moveit_config move_group.launch allow_active_sensing:=true` without specifying `moveit_octomap_sensor_params_file`